config/sprint_dates.csv

```diff
@@ -0,0 +1,5 @@
+﻿Sprint Name,Sprint Start Date
+Sprint week 1 and 2 2021,2021-01-04
+Sprint week 3 and 4 2021,2021-01-19
+Sprint week 5 and 6 2021,2021-02-01
+Sprint week 7 and 8 2021,2021-02-16

```

Comment:
I think I have managed to articulate why this bothered me: the title shouldn't be part of the input data. We should compute the title from the start date. This way we avoid silly typos.

Reply:
It does bother me as well. Great candidate for a future improvement.